src/Shared/ExceptionHandling.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 #if BUILDINGAPPXTASKS
+#nullable disable
+
```

Comment:
Just move this above the `#if`s?

Reply:
Fixed. Merged latest main and resolved conflict.